aten/src/ATen/Context.cpp

```diff
@@ -324,9 +324,9 @@ const std::vector<at::QEngine>& Context::supportedQEngines() {
 
 #ifdef USE_FBGEMM
     if (fbgemm::fbgemmSupportedCPU()) {
-      engines.push_back(at::kFBGEMM);
       // The X86 qengine is available if and only if FBGEMM is available
       engines.push_back(at::kX86);
+      engines.push_back(at::kFBGEMM);
```

Comment:
Just curious, order of supported engines is completely irrelevant here, isn't it?